doc/cli_vs_yum.rst

```diff
@@ -338,6 +338,7 @@ Original Yum tool          New DNF command                Package
 ``debuginfo-install``      `dnf debuginfo-install`_       ``dnf-plugins-core``
 ``find-repos-of-install``  `dnf list installed`_          ``dnf``
 ``needs-restarting``       `dnf tracer`_                  ``dnf-plugins-extras``
+``package-cleanup``        `dnf list`, `dnf repoquery`_   ``dnf-plugins-core``
```

Comment:
make a ref to info command e.g. :ref:`dnf info <info_command-label>` + create that label